docs/get-started/Velox.md

```diff
@@ -68,27 +68,25 @@ setting up build dependencies.
 ./dev/buildbundle-veloxbe.sh --enable_ep_cache=ON --build_arrow=OFF --run_setup_script=OFF
 ```
 
-**For aarch64 build:**
+**For aarch64 build**
 
 ```bash
 export CPU_TARGET="aarch64"
 
-./dev/builddeps-veloxbe.sh
+./dev/buildbundle-veloxbe.sh
 ```
 
-**Build Velox separately**
+**Step-by-step build**
 
+Alternative to the above one-step build, you can follow the below guide for step-by-step build.
 Currently, Gluten is using a [forked Velox](https://github.com/oap-project/velox/) which is daily updated based on [upstream Velox](https://github.com/facebookincubator/velox).
 
-Scripts under `/path/to/gluten/ep/build-velox/src` provide `get_velox.sh` and `build_velox.sh` to build Velox separately, you could use these scripts with custom repo/branch/location.
+```bash
 
-Velox provides arrow/parquet lib. Gluten cpp module need a required VELOX_HOME parsed by --velox_home, if you specify custom ep location, make sure these variables be passed correctly.
+./dev/builddeps-veloxbe.sh build_arrow
```

Comment:
Can you please make a note? 
we need this as we slightly modified arrow, but patches are merged into upstream already, so in the future we could use upstream arrow

Reply:
@zhouyuan, just updated. Thanks!